public/app/features/dashboard/components/DashNav/IirisMaintenance.tsx

```diff
@@ -295,25 +293,6 @@ class IirisMaintenance extends PureComponent<Props, State> {
       });
   };
 
-  /**
```

Comment:
Wiki-link is not relevant anymore?

Reply:
This was actually dead code. Wiki link is another component and separate file, IirisServiceInfoWikiButton.tsx.